server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -417,6 +480,16 @@ const accountMutations = {
         throw new Unauthorized();
       }
 
+      models.Activity.create({
+        type: activities.COLLECTIVE_EDITED,
+        UserId: req.remoteUser.id,
+        UserTokenId: req.userToken?.id,
+        CollectiveId: account.id,
+        FromCollectiveId: account.id,
+        HostCollectiveId: account.approvedAt ? account.HostCollectiveId : null,
+        data: { previousData: account.data?.policies, newData: args.policies },
+      });
+
```

Comment:
Can we do that after `setPolicies`? `setPolicies` can fail

Reply:
Done. 👍🏽